components/Details/Details.tsx

```diff
@@ -1,11 +1,21 @@
 import cn from "classnames";
 import { useContext, useEffect, useState, useMemo } from "react";
+import { useRouter } from "next/router";
 import HeadlessButton from "components/HeadlessButton";
 import Icon from "components/Icon";
 import { DocsContext, getScopes } from "layouts/DocsPage/context";
 import { ScopesType } from "layouts/DocsPage/types";
+import { getAnchor } from "utils/url";
 import styles from "./Details.module.css";
 
+const transformTitleToAnchor = (title: string): string => {
+  return title
+    .replace(/[&\/\\#,+()$~%.'":*?<>{}^;\d]/g, "")
+    .replace(/\s/g, "-")
+    .replace(/-$/, "")
+    .toLowerCase();
+};
```

Comment:
FYI: There is a library from github that do exactly that and also works with non-latin symbols and solves the problem with multiple headers with the same text on the page: https://github.com/Flet/github-slugger

For example `rehype-slug` uses it inside. It's probably better to use this library on the server through.

Not to rewrite it right now, but if we have some of these problems in the future.